core/jazz_ui/src/app/pages/service-detail/service-detail.component.ts

```diff
@@ -219,7 +224,12 @@ export class ServiceDetailComponent implements OnInit {
     this.isLoadingService = true;
     this.http.get('/jazz/services/' + id, null, id).subscribe(response => {
       let service = response.data;
-      this.provider = response.data.deployment_accounts[0].provider
+      if(response.data.deployment_accounts){
+        this.provider = response.data.deployment_accounts[0].provider
+      } else {
+        // core services will always be in aws
+        this.provider = 'aws'
```

Comment:
Let core services specify provider value during service creation. Let's avoid these assumptions if possible. Remove `if.. else` block.

Reply:
Since core services doesn't have the required deployment_accounts field, we might need to make other changes for specifying the provider for core services during stack installation

Reply:
Yes, we need this change at the installer side (if not being done already).

Reply:
Done. Removed the if else block. Services are expected to have now deployment_accounts field